maggy/core/environment/databricks.py

```diff
@@ -0,0 +1,66 @@
+#
+#   Copyright 2021 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import os
+
+from maggy.core.environment.base import BaseEnv
+
+
+class DatabricksEnv(BaseEnv):
+    """
+    This class extends BaseEnv.
+    Environment implemented for maggy usage on Databricks.
+    """
+
+    def __init__(self):
+        self.log_dir = "/dbfs/maggy_log/"
```

Comment:
```suggestion
    def __init__(self):
        super().__init__()
        self.log_dir = "/dbfs/maggy_log/"
```
Super init is missing. Were the functions from BaseEnv working? This should not be possible.